apps/teamsjs-diagnostic-app/env/.env.dev

```diff
@@ -0,0 +1,23 @@
+# This file includes environment variables that will be committed to git by default.
+
+# Built-in environment variables
+TEAMSFX_ENV=dev
+APP_NAME_SUFFIX=dev
+# Updating AZURE_SUBSCRIPTION_ID or AZURE_RESOURCE_GROUP_NAME after provision may also require an update to RESOURCE_SUFFIX, because some services require a globally unique name across subscriptions/resource groups.
+AZURE_SUBSCRIPTION_ID=
```

Comment:
Can you say more @shrshindeMSFT ? Are you implying that these shouldn't be here at all, or that they should have values set?

Reply:
@TrevorJoelHarris This is something which should reside in environment and not to a specific app. From my understanding, it could lead to major security concerns exposing the IDs here. For example, if someone gets access to this Subscription ID, what all things that person can perform, also point to note would be attacks don't only happen from outside.

Reply:
That makes sense! I deleted it!